mephisto/scripts/local_db/gh_actions/auto_generate_blueprint.py

```diff
@@ -20,6 +22,14 @@ def create_blueprint_info(blueprint_file, arg_dict):
                 if isinstance(item_content, str)
                 else item_content
             )
+            if (
+                isinstance(item_to_append, str)
+                and item_to_append.find("mephisto/") != -1
+            ):
+                item_to_append = item_to_append[
+                    item_to_append.find("mephisto/") : len(item_to_append)
```

Comment:
this is not very robust. this is obviously a contrived example, but a user can have their code located somewhere like this:

> /User/mephisto/my documents/mephisto/private secret folder/mephisto

The result of this would yield back:

> my documents/mephisto/private secret folder/mephisto

This is why I suggested "subtracting out the root folder" before